### PR TITLE
hillel & noam - wrote intake commands

### PR DESCRIPTION
**Pull request by greenblitz4590**

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/subsystems/Intake/IntakeConstants.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 public class IntakeConstants {
 	public static final int INTAKE_ID = 0;
+	public static final double POWER_TO_RUN = 0;
 	
 	public static class SimulationConstants {
```
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/robotName/subsystems/Intake/IntakeConstants.java:9]: make SimulationConstants in a different file like in the SwerveModules subsystem

File: src/main/java/edu/greenblitz/robotName/commands/intake/ReverseRunIntake.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package edu.greenblitz.robotName.commands.intake;
+
+import edu.greenblitz.robotName.subsystems.Intake.IntakeConstants;
+
+public class ReverseRunIntake extends IntakeCommand{
+    @Override
+    public void execute(){
+        intake.setPower(-IntakeConstants.POWER_TO_RUN);
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/commands/intake/ReverseRunIntake.java:8]: just save a POWER_TO_EJECT or REVERSE_RUN_POWER

**Review by bossyshrimp68 — CHANGES_REQUESTED** [review]

File: src/main/java/edu/greenblitz/robotName/commands/intake/ReverseRunIntake.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package edu.greenblitz.robotName.commands.intake;
+
+import edu.greenblitz.robotName.subsystems.Intake.IntakeConstants;
+
+public class ReverseRunIntake extends IntakeCommand{
+    @Override
+    public void execute(){
+        intake.setPower(IntakeConstants.POWER_TO_EJECT);
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/commands/intake/ReverseRunIntake.java:8]:
```suggestion
        intake.setPower(IntakeConstants.POWER_TO_REVERSE_RUN);
```
mb

**Review by bossyshrimp68 — CHANGES_REQUESTED** [review]

**Review by bossyshrimp68 — APPROVED** [review]
gg

**Review by bossyshrimp68 — APPROVED** [review]

**Review by tnrz135 — APPROVED** [review]

**Pull request merged**